packages/wonder-blocks-modal/src/components/__tests__/modal-launcher.test.js

```diff
@@ -289,45 +294,113 @@ describe("ModalLauncher", () => {
 
     test("if modal is closed, return focus to the last element focused outside the modal", async () => {
         // Arrange
-        // We need the elements in the DOM document, it seems, for this test
-        // to work. Changing to testing-library will likely fix this.
-        const containerDiv = getElementAttachedToDocument("container");
         let savedCloseModal = () => {
             throw new Error(`closeModal wasn't saved`);
         };
 
-        const wrapper = mount(
-            <ModalLauncher
-                modal={({closeModal}) => {
-                    savedCloseModal = closeModal;
-                    return exampleModal;
-                }}
-            >
-                {({openModal}) => (
-                    <button onClick={openModal} data-last-focused-button />
-                )}
-            </ModalLauncher>,
-            {attachTo: containerDiv},
+        render(
+            <View>
+                <Button>Not last element button</Button>
+                <ModalLauncher
+                    modal={({closeModal}) => {
+                        savedCloseModal = closeModal;
+                        return exampleModal;
+                    }}
+                >
+                    {({openModal}) => (
+                        <button
+                            onClick={openModal}
+                            data-test-id="data-last-focused-button"
+                        />
+                    )}
+                </ModalLauncher>
+            </View>,
         );
 
-        const lastButton = wrapper
-            .find("[data-last-focused-button]")
-            .getDOMNode();
-        // force focus
-        lastButton.focus();
+        const lastButton = await screen.findByTestId(
+            "data-last-focused-button",
+        );
 
         // Launch the modal.
-        wrapper.find("button").simulate("click");
-
-        // wait for styles to be applied
-        await wait();
+        userEvent.click(lastButton);
 
         // Act
         savedCloseModal(); // close the modal
-        wrapper.update();
 
         // Assert
-        expect(document.activeElement).toBe(lastButton);
+        await waitFor(() => {
+            expect(lastButton).toHaveFocus();
+        });
+    });
+
+    test("if `closedFocusId` is passed, shift focus to specified element after the modal closes", async () => {
+        // Arrange
+        const ModalLauncherWrapper = () => {
+            const [opened, setOpened] = React.useState(false);
+
+            const handleOpen = () => {
+                console.log("opening modal");
+                setOpened(true);
+            };
+
+            const handleClose = () => {
+                console.log("closing modal");
+                setOpened(false);
+            };
+
+            return (
+                <View style={{gap: 20}}>
```

Comment:
Maybe this is copypasta from the stories :) Yo won't need to use `style` on tests, unless you are testing something related to styling (that is usually tested with Chromatic) :).